util/chunk/column.go

```diff
@@ -334,6 +334,14 @@ func (c *Column) SetNulls(begin, end int, isNull bool) {
 	}
 }
 
+func (c *Column) SetBytes(rowID int, bs []byte) {
```

Comment:
Could we name it as `SetRaw`? Since here's a `GetRaw` method.
And do not assume that the underlying data is fixed-size.